easybuild/tools/utilities.py

```diff
@@ -160,3 +162,12 @@ def error(*args, **kwargs):
             return error
 
     return wrap
+
+
+def trace_msg(message, silent=False, timestamp=False):
+    """Print trace message."""
+    if build_option('trace'):
+        _log.experimental("Using --trace")
+        if timestamp:
+            message += " [started at: %s]" % datetime.now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Don't your normally put the time first and then the message?

Reply:
Yeah, I've tried both approaches, not happy with either (because of the `>>`). I'm open to suggestions.

I'm up for reformatting this, but we can do that later since `--trace` requires `--experimental` for now; I want to avoid this being blocked by a bike-shedding discussion and get it merged ASAP to be included in EB v3.4.0...